index.md

```diff
@@ -19,4 +19,5 @@ architecture/proposed/long-term-plans.md
 security-advisories/security-advisories.md
 maintainers.md
 CONTRIBUTING.md
+kb/index.md
```

Comment:
The paths to the old knowledge base were e.g. `https://tls.mbed.org/kb/how-to/add-a-random-generator` with no extension. Now the articles are at e.g. `https://tls.mbed.org/kb/how-to/add-a-random-generator.html`, i.e. with `.html` at the end. So existing links still don't work. But at least the articles are available.

Reply:
We can handle this in two ways - the sphinx build corresponding to the old naming scheme is `dirhtml`, instead of the `html` we are currently using.

Readthedocs also supports adding redirections between the two schemes automatically.
So we can switch to the old scheme (and redirect <foo>.html to <foo>), or just switch on redirections from <foo> to <foo>.html.

Both options are fairly easy to achieve.

Reply:
https://github.com/Mbed-TLS/mbedtls-docs/pull/40 removes the `.html` extension from the new site.

This is only part of the solution because the current redirection on `tls.mbed.org` redirects all pages to `https://www.trustedfirmware.org/projects/mbed-tls/`. We need to change this to a redirection that remembers the source and redirects under `https://mbed-tls.readthedocs.io/en/latest/` for most URLs.